wolfcrypt/src/rsa.c

```diff
@@ -3106,7 +3106,7 @@ int wc_RsaPrivateDecrypt_ex(const byte* in, word32 inLen, byte* out,
 #endif /* WC_NO_RSA_OAEP || WC_RSA_NO_PADDING */
 #endif /* WOLFSSL_RSA_PUBLIC_ONLY */
 
-
+#if !defined(WOLFSSL_CRYPTOCELL)
```

Comment:
Disabling this function here is going to cause link errors. Typically we would keep the `wc_RsaSSL_VerifyInline` function, but disable its content and return an error code like `NOT_COMPILED_IN`.

Reply:
I think it is better to cause a link error for Cryptocell build to make sure a user knows it is unsupported.  Did I miss something?

Reply:
Then it would be good to also disable any wolf code that calls this so our library still builds.

Reply:
@dgarske, The library still builds with or without WOLFSSL_CRYPTOCELL defined except the wc_RsaSSL_VerifyInline() would be unavailable when using WOLFSSL_CRYPTOCELL.  When porting an application with wc_RsaSSL_VerifyInline() with CRYPTOCELL, you will get a linker error. This way a user would get the error early instead of something going wrong during runtime.